### PR TITLE
[l2-load-balancer][docs] Add visualisations to illustrate the differences between MetalLB+L2 and L2LoadBalancer

### PR DESCRIPTION
**Pull request by Beastlex**
## Description
Add visualisations to illustrate the differences between MetalLB+L2 and L2LoadBalancer.

## Why do we need it, and what problem does it solve?
Clarifications...

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: l2-load-balancer
type: chore
summary: Added visualisations to illustrate the differences between MetalLB+L2 and L2LoadBalancer.
impact_level: low
```

### CODE REVIEW

File: ee/modules/381-l2-load-balancer/docs/README_RU.md [hunk 1]
```diff
@@ -10,11 +10,17 @@ title: "Модуль l2-load-balancer"
 
 [[MetalLB в режиме L2]](../../modules/380-metallb/#режим-layer-2) позволяет заказать _Service_ с типом `LoadBalancer`, работа которого основана на том, что в пиринговой сети узлы-балансировщики имитируют ARP-ответы от "публичного" IP. Данный режим имеет существенное ограничение — единовременно лишь один балансировочный узел обрабатывает весь входящий трафик данного сервиса. Соответственно:
 
+<div data-presentation="../../presentations/381-l2-load-balancer/basics_metallb_ru.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/1cs1uKeX53DB973EMtLFcc8UQ8BFCW6FY2vmEWua1tu8/ --->
+
 * Узел, выбранный в качестве лидера для "публичного" IP становится "узким местом" без возможности горизонтального масштабирования.
 * В случае выхода узла-балансировщика из строя, все текущие соединения будут оборваны для переключения на новый узел, который будет назначен лидером.
```
Comment (apolovov) [inline ee/modules/381-l2-load-balancer/docs/README_RU.md:18]: Picture here pls.

File: ee/modules/381-l2-load-balancer/docs/README_RU.md [hunk 2]
```diff
@@ -10,11 +10,17 @@ title: "Модуль l2-load-balancer"
...
 
 Данный модуль помогает обойти эти ограничения. Он предоставляет новый интерфейс _L2LoadBalancer_, который:
 
+<div data-presentation="../../presentations/381-l2-load-balancer/basics_l2loadbalancer_ru.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/1mDfYi2E2tUni5ReQbXvA9CAZBsHXSGynArw9YB6VGvM/ --->
+
 * Позволяет автоматически заказать несколько _Service_ с типом `LoadBalancer` по количеству узлов-балансировщиков.
 * Обеспечивает равномерное распределение L2-лидерства между узлами-балансировщиками.
```
Comment (apolovov) [inline ee/modules/381-l2-load-balancer/docs/README_RU.md:26]:
```suggestion
[[MetalLB в режиме L2]](../../modules/380-metallb/#режим-layer-2) позволяет заказать _Service_ с типом `LoadBalancer`, работа которого основана на том, что в пиринговой сети узлы-балансировщики имитируют ARP-ответы от "публичного" IP. Данный режим имеет существенное ограничение — единовременно лишь один балансировочный узел обрабатывает весь входящий трафик данного сервиса. Соответственно:

* Узел, выбранный в качестве лидера для "публичного" IP становится "узким местом" без возможности горизонтального масштабирования.
* В случае выхода узла-балансировщика из строя, все текущие соединения будут оборваны для переключения на новый узел, который будет назначен лидером.

<div data-presentation="../../presentations/381-l2-load-balancer/basics_metallb_ru.pdf"></div>
<!--- Source: https://docs.google.com/presentation/d/1cs1uKeX53DB973EMtLFcc8UQ8BFCW6FY2vmEWua1tu8/ --->

Данный модуль помогает обойти эти ограничения. Он предоставляет новый интерфейс _L2LoadBalancer_, который:

* Позволяет автоматически заказать несколько _Service_ с типом `LoadBalancer` по количеству узлов-балансировщиков.
* Обеспечивает равномерное распределение L2-лидерства между узлами-балансировщиками.

<div data-presentation="../../presentations/381-l2-load-balancer/basics_l2loadbalancer_ru.pdf"></div>
<!--- Source: https://docs.google.com/presentation/d/1mDfYi2E2tUni5ReQbXvA9CAZBsHXSGynArw9YB6VGvM/ --->
```

File: ee/modules/381-l2-load-balancer/docs/README.md [hunk 3]
```diff
@@ -10,11 +10,17 @@ Based on [MetalLB](https://metallb.universe.tf/) solution.
...
 * The node selected as the leader for the "public" IP becomes a "bottleneck" with no possibility of horizontal scaling.
 * If the balancer node fails, all current connections will be dropped for switching to a new balancing node that will be selected as the leader.
 
 This module helps bypass these limitations. It provides a new _L2LoadBalancer_ interface that:
 
+<div data-presentation="../../presentations/381-l2-load-balancer/basics_l2loadbalancer_en.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/1rVQ2RZ645HKAcMFU2hI_MPJZu_zA70IZ2OYWI1wSW34/ --->
+
 * Allows to automatically order multiple _Service_ with type `LoadBalancer` according to the number of balancer nodes.
 * Provides an even distribution of L2 leadership among the balancer nodes.
```
Comment (apolovov) [inline ee/modules/381-l2-load-balancer/docs/README.md:26]:
```suggestion
[[MetalLB in L2 mode]](../../modules/380-metallb/#layer-2-mode) allows to order _Service_ with `LoadBalancer` type, the operation of which is based on the fact that balancing nodes simulate ARP-responses from the "public" IP in a peering network. This mode has a significant limitation — only one balancing node handles all the incoming traffic of this service at a time. Therefore:

* The node selected as the leader for the "public" IP becomes a "bottleneck" with no possibility of horizontal scaling.
* If the balancer node fails, all current connections will be dropped for switching to a new balancing node that will be selected as the leader.

<div data-presentation="../../presentations/381-l2-load-balancer/basics_metallb_en.pdf"></div>
<!--- Source: https://docs.google.com/presentation/d/18vcVJ1cY2yn19vBM_dTNW3hF0w9SE4S81VZc2P6fVFM/ --->

This module helps bypass these limitations. It provides a new _L2LoadBalancer_ interface that:

* Allows to automatically order multiple _Service_ with type `LoadBalancer` according to the number of balancer nodes.
* Provides an even distribution of L2 leadership among the balancer nodes.

<div data-presentation="../../presentations/381-l2-load-balancer/basics_l2loadbalancer_en.pdf"></div>
<!--- Source: https://docs.google.com/presentation/d/1rVQ2RZ645HKAcMFU2hI_MPJZu_zA70IZ2OYWI1wSW34/ --->
```

File: ee/modules/381-l2-load-balancer/docs/README_RU.md [hunk 4]
```diff
@@ -13,11 +13,16 @@ title: "Модуль l2-load-balancer"
...
+<!--- Source: https://docs.google.com/presentation/d/1cs1uKeX53DB973EMtLFcc8UQ8BFCW6FY2vmEWua1tu8/ --->
+
 Данный модуль помогает обойти эти ограничения. Он предоставляет новый интерфейс _L2LoadBalancer_, который:
 
 * Позволяет автоматически заказать несколько _Service_ с типом `LoadBalancer` по количеству узлов-балансировщиков.
 * Обеспечивает равномерное распределение L2-лидерства между узлами-балансировщиками.
 
+<div data-presentation="../../presentations/381-l2-load-balancer/basics_l2loadbalancer_ru.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/1mDfYi2E2tUni5ReQbXvA9CAZBsHXSGynArw9YB6VGvM/ --->
 Таким образом:
```
Comment (apolovov) [inline ee/modules/381-l2-load-balancer/docs/README_RU.md:26]:
```suggestion

Таким образом:
```

File: ee/modules/381-l2-load-balancer/docs/README.md [hunk 5]
```diff
@@ -13,11 +13,16 @@ Based on [MetalLB](https://metallb.universe.tf/) solution.
...
+<!--- Source: https://docs.google.com/presentation/d/18vcVJ1cY2yn19vBM_dTNW3hF0w9SE4S81VZc2P6fVFM/ --->
+
 This module helps bypass these limitations. It provides a new _L2LoadBalancer_ interface that:
 
 * Allows to automatically order multiple _Service_ with type `LoadBalancer` according to the number of balancer nodes.
 * Provides an even distribution of L2 leadership among the balancer nodes.
 
+<div data-presentation="../../presentations/381-l2-load-balancer/basics_l2loadbalancer_en.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/1rVQ2RZ645HKAcMFU2hI_MPJZu_zA70IZ2OYWI1wSW34/ --->
 Thus:
```
Comment (apolovov) [inline ee/modules/381-l2-load-balancer/docs/README.md:26]:
```suggestion

Thus:
```

**Review by apolovov — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**